### DISCUSSION TITLE
夜间模式部分页面中留有一些白底

### DISCUSSION THREAD

**Discussion by xuwenchao980507** [the1812/Bilibili-Evolved#4344] [功能建议 / Ideas]
作者请问在这个B站番剧播放器页面中，夜间模式还有部分白底无法消除，播放器两边、番剧简介、以及番剧推荐都还是白的。后续能再优化下吗？感谢
![微信截图_20230726111100](https://github.com/the1812/Bilibili-Evolved/assets/95979459/c514845d-1792-4f67-8252-870c9f53c445)

**Comment by Geeyun-JY3** [comment]
请善用搜索，新版番剧播放页还没适配：#4186
